datasketch/minhash.py

```diff
@@ -168,13 +217,19 @@ def _hash_32(b):
         phv = np.bitwise_and((a * hv + b) % _mersenne_prime, _max_hash)
         self.hashvalues = np.minimum(phv, self.hashvalues)
 
-    def update_batch(self, b: Iterable) -> None:
+    def update_batch(self, b: Iterable, *, use_gpu: Optional[bool] = None) -> None:
         """Update this MinHash with new values.
         The values will be hashed using the hash function specified by
         the `hashfunc` argument in the constructor.
 
+        If GPU is enabled globally (via constructor or enable_gpu()) or per-call
```

Comment:
I don't think we should have this override, it creates too much flexibility that leads to more ways of writing the code, and more room for errors.